apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveService.kt

```diff
@@ -494,21 +484,16 @@ class OppgaveService(
         return oppgaveDao.hentOppgave(oppgaveLagres.id)!!
     }
 
-    fun hentOppgave(oppgaveId: UUID): OppgaveIntern? {
+    fun hentOppgave(oppgaveId: UUID): OppgaveIntern {
         return oppgaveDao.hentOppgave(oppgaveId)
+            ?: throw InternfeilException("Oppgave med id=$oppgaveId ikke funnet!")
```

Comment:
mulig jeg ikke helt ser bruken andre steder, men jeg synes retur av nullable oppgave dokumenterer tydeligere at det er mulighet for at man ikke finner en oppgave, vs. denne der man må lese funksjonen for å se at den kaster en feil. selvfølgelig, hvis man tenker litt over hva man gjør så er det åpenbart at det kan være at vi ikke har en match på en gitt id når man bruker denne funksjonen andre steder -- men virker ikke så ille å bare kommunisere det eksplisitt i typen.

Reply:
Helt enig, men det ble gjort slik for å støtte gammel moro. Skal fjernes etter hvert. 

Jeg mener selv at henting på ID _alltid_ skal kaste feil dersom noe ikke finnes, siden det åpenbart er en feil. Hvorfor ellers har man ID-en. 

Kommer en hel del opprydding når endringene har fått kjøre litt i prod og blitt bekreftet OK.